infrastructure/dogfood/terraform/gcp/loadbalancer.tf

```diff
@@ -1,5 +1,5 @@
 resource "google_dns_managed_zone" "default" {
-  dns_name = var.dns_name
+  dns_name = var.dns_zone
```

Comment:
This seems to create a potential break when its referenced in loadbalancer.tf:25

Reply:
I would just remove the dns_zone variable since its unused.

Reply:
> I would just remove the dns_zone variable since its unused.

It _should_ be used otherwise the SOA/NS and A records are for the same FQDN. If it makes sense to get rid of `dns_zone` then it should get derived from `dns_name`, but setting `dns_name` to the same value for both resources does not seem functionally right.